config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -54,8 +54,8 @@ presubmits:
             cpu: "2"
             memory: "6Gi"
 
-  # This jobs runs e2e.test with a focus on tests for all alpha storage features on a kind cluster
-  - name: pull-kubernetes-e2e-storage-kind-alpha-features
+  # This jobs runs e2e.test with a focus on tests for all alpha/beta storage features on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-features
```

Comment:
I don't know if is better to reflect the alpha-beta in the job name instead of removing it, up to you, for me is easier to understand ` pull-kubernetes-e2e-storage-kind-alpha-beta-features`

Reply:
renamed.